lib/pages/new_group/new_group_info_controller.dart

```diff
@@ -14,18 +14,23 @@ extension NewGroupInfoController on NewGroupController {
     });
   }
 
-  void moveToGroupChatScreen() async {
-    final roomId = await Matrix.of(context).client.createGroupChat(
+  void moveToGroupChatScreen(MatrixFile? avatar) async {
+    final client = Matrix.of(context).client;
+    final roomId = await client.createGroupChat(
       groupName: groupName,
       invite: getSelectedValidContacts(contactsList)
         .map<String>((contact) => contact.matrixId!)
         .toList(),
       enableEncryption: isEnableEEEncryptionNotifier.value,
-      preset: isGroupPrivate 
+      preset: isGroupPrivate
         ? CreateRoomPreset.privateChat
         : CreateRoomPreset.publicChat,
     );
 
+    if (roomId.isNotEmpty && avatar != null) {
+      await client.getRoomById(roomId)!.setAvatar(avatar);
+    }
+
```

Comment:
the error handling is not good in current base. So please think more about it:
- what happen if `createGroup` failed?
- what happen if `createGroup` success but `set avatar` failed?